src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/AggregateWorkspaceProjectContext.cs

```diff
@@ -51,7 +51,7 @@ internal sealed class AggregateWorkspaceProjectContext
 
         public object ENCProjectConfig => _configuredProjectContextsByTargetFramework[_activeTargetFramework];
 
-        public bool IsCrossTargeting => _activeTargetFramework.Length > 0;
+        public bool IsCrossTargeting => _configuredProjectContextsByTargetFramework.Count > 0;
```

Comment:
For cross targeting projects, wouldn't this be 2 or more target frameworks?

Reply:
Yes.  Yes it would.  Good catch!